### PR TITLE
Add pageHeader.pad theme documentation

### PR DESCRIPTION
**Pull request by halocline**
Documents `pageHeader.pad`

Related to and dependent upon https://github.com/grommet/grommet/pull/6152 (merged)

Related to https://github.com/grommet/hpe-design-system/issues/2603

### CODE REVIEW

File: src/screens/PageHeader.js [hunk 1]
```diff
@@ -154,6 +157,16 @@ const PageHeaderPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="pageHeader.pad">
+          <Description>Padding applied to PageHeader.</Description>
+          <PropertyValue type="string">
```
Comment (taysea) [inline src/screens/PageHeader.js:162]: Is there a default value?

Comment (halocline) [reply src/screens/PageHeader.js:162]: There is now. :)

**Review by taysea — APPROVED** [review]

**Review by jcfilben — COMMENTED** [review]
Minor comment, otherwise looks good

**Review by jcfilben — APPROVED** [review]
Looks good!

**Pull request merged**